templates/minio_compose.j2

```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+#networks:
+#  {{ minio.docker.network.name }}:
+#    driver: bridge
+#    external: {{ minio.docker.network.external }}
+
+services:
+  {{ minio.docker.service_name }}:
+    container_name: {{ minio.docker.service_name }}
+    image: quay.io/minio/minio:{{ minio.docker.version }}
+    environment:
+        TZ: {{ minio.docker.timezone }}
+        MINIO_ROOT_USER: {{ minio.docker.root_user }}
+        MINIO_ROOT_PASSWORD: {{ minio.docker.root_password }}
+        MINIO_ACCESS_KEY: {{ minio.docker.access_key }}
+        MINIO_SECRET_KEY: {{ minio.docker.secret_key }}
+    ports:
```

Comment:
As a future optimization, we might want to remove MINIO_ACCESS_KEY and MINIO_ACCESS_KEY.

In https://github.com/IQSS/dataverse/pull/10044 I'm only using MINIO_ROOT_USER and MINIO_ROOT_PASSWORD. These env vars are less intuitive to me but at least they aren't deprecated. See https://min.io/docs/minio/linux/administration/identity-access-management/minio-user-management.html#minio-root-user

![Screenshot 2023-12-06 at 4 07 42 PM](https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/assets/21006/66ef3165-cadb-468a-8a73-f4a24b088448)